cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -98,15 +110,38 @@ def _init_options(self, kwargs):
 
         self.options.setdefault("sources", [])
 
+    def determine_output_directory(self):
+
+        # This code is not clear to me so I'm a bit afraid to change it
+        def relativize_path(output_dir):
+            return os.path.relpath(
+                os.path.join(self.working_path, output_dir), os.getcwd()
+            )
+
+        output_dir = None
+
+        if os.environ.get("CUMULUSCI_ROBOT_OUTPUT_DIR"):
+            output_dir = relativize_path(os.environ.get("CUMULUSCI_ROBOT_OUTPUT_DIR"))
+
+        fallback_paths = [f"robot/{self.project_config.repo_name}/results", "."]
+
+        while not output_dir:
```

Comment:
I don't like this behavior. If I set the outputdir to robot/output but robot/output doesn't exist, this will end up dumping everything in the current directory, right? This is different than robot's own default which will create the path if it doesn't exist. I think we should accept whatever is given, and let robot create the folder if it doesn't exist.